DEVELOPING.md

```diff
@@ -23,23 +23,11 @@ export INCIDENT_API_KEY="inc_development_<token>"
 
 This points the provider at your local instance via ngrok.
 
-If you need to regenerate the client, you first need to copy the following files over from `core/server/api/public/gen/http/`:
-
-```
-openapi.json
-openapi.yaml
-openapi3-secret.json
-openapi3.json
-openapi3.yaml
-```
+If you need to regenerate the client, you first need to copy the following files from `core`:
```

Comment:
missing a file in here?

Reply:
no I just forgot to change the pluralisation. It's just one file now!